docs/commands/zq.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 0
+sidebar_position: 1
 sidebar_label: zq
 description: A command-line tool that uses the Zed Language for pipeline-style search and analytics.
```

Comment:
It's unclear to me from the Docusaurus docs how this `description` field is actually used. I don't think it's populating anything reader-facing for us at the moment so I was tempted to remove it, but I decided it's not hurting anything so I left it alone.